### PR TITLE
Adding new methods for storage

### PR DESCRIPTION
**Pull request by Benkdel**
Please help me review this pull request, thank you

### CODE REVIEW

**Review by CarlosGBentinDN — CHANGES_REQUESTED** [review]
please review line 84 of file_storage.py, there is no space btw variable and =

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,22 @@ def delete(self, obj=None):
...
+           type(id) is str and cls in classes:
+            key = cls + '.' + id
+            obj = self.__objects.get(key, None)
+            return obj
+        else:
+            return None
+
+    def count(self, cls=None):
+        """ Count number of objects in storage """
+        total= 0
```
Comment (humtej1204) [inline models/engine/file_storage.py:84]: You have to add a space before the symbol "=" to could pass the pycodestyle

**Review by humtej1204 — COMMENTED** [review]
You need to check your pycodestyle.

**Review by humtej1204 — APPROVED** [review]
The Pycodestyle seems to be OK. No more errors to fix.

**Pull request merged**